samples/03_org_administrators/validate_item_metadata.ipynb

```diff
@@ -0,0 +1,385 @@
+{
```

Comment:
<p>We can also add a script to delete the item at the end of this notebook (though it doesn't create any service)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1566/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Thanks @cariashuang0417 . I added the last `item.delete()` cell with a markdown before it to explain it.